CONTRIBUTING.md

```diff
@@ -12,4 +12,20 @@ If you like our project, but just don’t have time to contribute, that’s fine
 * Tweet about it and also check out our [twitter page](https://twitter.com/WildFlyAS) and [mastodon page](https://fosstodon.org/@wildflyas).
 * Check out our [youtube](https://www.youtube.com/@WildFlyAS) contents.
 
+# Issues
+
+---
+
+WildFly uses JIRA to manage issues. All issues can be found [here](https://issues.redhat.com/projects/WFLY/issues/).
+
+To create a new issue, comment on an existing issue, or assign an issue to yourself, you'll need to first [create a JIRA account](https://issues.redhat.com/).
+
+# Good First Issues
+
+---
+
+Check out our issues with the `good-first-issue` label. These are a triaged set of issues that are great for getting started on our project. These can be found [here](https://issues.redhat.com/issues/?filter=12403174).
+
+Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. To assign an issue to yourself, simply click on "Start Progress". This will automatically assign the issue to you.
```

Comment:
Can you add this sentence:

If you're not able to assign it to yourself that way, post a message in [the wildfly-developers Zulip stream](https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers) and someone will help get it assigned to you.

Reply:
@bstansberry Implemented it.